{"uuid":"55dcc214-ea56-4524-835f-a2590353f66b","historyId":"f407fce0a57809c6d55e087a9423daf2","fullName":"Tests.ChangePassPageTest.TC_PSA_0076","labels":[{"name":"package","value":"Tests.ChangePassPageTest"},{"name":"testClass","value":"Tests.ChangePassPageTest"},{"name":"testMethod","value":"TC_PSA_0076"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"SAPortalTest_Selenium_POM"},{"name":"subSuite","value":"Tests.ChangePassPageTest"},{"name":"host","value":"DESKTOP-MCJ49E3"},{"name":"thread","value":"6572@DESKTOP-MCJ49E3.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Validation of login with new password","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tError:Clicking on the Login button doesn't navigate to the right page\nExpected URL : http://20.74.237.100:3002/otp\nActual URL :http://20.74.237.100:3002/login expected [true] but found [false],\n\tError:Error Message appears when login with the new pass ","trace":"java.lang.AssertionError: The following asserts failed:\n\tError:Clicking on the Login button doesn't navigate to the right page\nExpected URL : http://20.74.237.100:3002/otp\nActual URL :http://20.74.237.100:3002/login expected [true] but found [false],\n\tError:Error Message appears when login with the new pass \r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)\r\n\tat Tests.ChangePassPageTest.TC_PSA_0076(ChangePassPageTest.java:1113)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1667382392825,"stop":1667382402923}